ppx_deriving.opam

```diff
@@ -19,10 +19,10 @@ depends: [
   "ppxfind"  {build}
   "ocaml-migrate-parsetree"
   "ppx_derivers"
-  "ppxlib"  {>= "0.9.0"}
+  "ppxlib" {>= "0.9.0"}
   "result"
-  "ounit"      {with-test}
-  "ocaml" {>= "4.04.1"}
+  "ounit" {with-test}
+  "ocaml" {>= "4.05.0"}
```

Comment:
Why should we reject OCaml 4.04? The current code looks actually to work fine on 4.04 (even on 4.04.0). Anyway, if we want to constrain users to use OCaml 4.05, I think we should update the synopsis as well.